src/fairseq2/models/s2t_transformer/builder.py

```diff
@@ -90,6 +90,9 @@ class S2TTransformerConfig:
     depthwise_conv_kernel_size: int
     """The kernel size of depthwise convolutions in Conformer blocks."""
 
+    depthwise_conv_is_causal: bool
```

Comment:
S2T Transformer architecture has never supported causal depthwise convolution. We should not expose this as a configurable parameter in S2T Transformer.